crates/core/thin_edge_json/src/event.rs

```diff
@@ -39,27 +42,47 @@ pub mod error {
 
         #[error(transparent)]
         SerdeJsonError(#[from] serde_json::error::Error),
+
+        #[error(transparent)]
+        JsonWriterError(#[from] JsonWriterError),
```

Comment:
```suggestion
        JsonWriterError(#[from] json_writer::JsonWriterError),
```

Then, you can remove the line 33,
```rust
use json_writer::JsonWriterError;
```